dnf/conf/config.py

```diff
@@ -429,6 +429,8 @@ def __init__(self, parent, section=None, parser=None):
         super(RepoConf, self).__init__(libdnf.conf.ConfigRepo(
             parent._config if parent else libdnf.conf.ConfigMain()), section, parser)
         self._masterConfig = parent._config if parent else libdnf.conf.ConfigMain()
+        if section:
+            self.name = section
```

Comment:
I am looking in libdnf. Please what represents in class ConfigRepo : public Config() `repo.name` and `repo.id`?

Reply:
Class libdnf::ConfigRepo contains only `name`. `id` is stored in class libdnf::Repo.
But in DNF (Python) are both elements directly accesible in Repo class. 
The `id` represents section name in config INI file. It must be unique. No more repos can have the same id.
The `name` represent key `name` in config INI file. I think it is more human readable name/description.